README.md

```diff
@@ -67,6 +67,8 @@ Options:
   -g, --globalConfig [path]  Global Test data config folder relative/ absolute path.  e.g: <path>/global-config
   -u, --url [url]          Server URL. e.g: http://localhost:9000
   -r, --report <n>         Will generate the html report or not. Default 0; e.g: 1 or 0
+  -i, --inline-report-assets <n> Inline assets in generated HTML report.
```

Comment:
Some Jenkins (CI-CD) are unable to load assets from a different files, so the option to generate HTML with inline assets